packages/manager/apps/container/src/container/legacy/server-sidebar/universe/public-cloud/pci-menu.ts

```diff
@@ -49,382 +47,444 @@ export const features = [
   'public-cloud:project-settings',
   'ai-dashboard',
   'data-platform',
-  'ai-endpoints'
+  'ai-endpoints',
+  'key-management-service'
 ];
 
 export function getPciProjectMenu(
   projectId: string,
   region: string,
+  featureAvailabilities: Record<string, boolean>,
   getURL: (app: string, path: string) => string,
 ): PciMenuItem[] {
-  return [
-    {
+  const pciMenu: PciMenuItem[] = [];
+
+  const isFeaturesAvailable = (...features: string[]): boolean =>
+    Boolean(features.find((feature: string) => featureAvailabilities[feature]));
```

Comment:
If I understand well, if you want to check if there is at least one feature available, I think it could be more efficient to prevent from iterating on the whole list by using `some` instead of `find`: 
```suggestion
   const isFeaturesAvailable = (...features: string[]): boolean =>
    features.some((feature: string) => featureAvailabilities[feature])
```